src/techs/techs.service.ts

```diff
@@ -136,14 +161,15 @@ export class TechsService {
                     name: createTechVoteDto.techName,
                     categoryId: createTechVoteDto.techCategoryId,
                     voyageTeamId: teamId,
+                    voyageTeamMemberId: createTechVoteDto.voyageTeamMemberId,
```

Comment:
I'm getting an error saying "Object literal may only specify known properties, but 'voyageTeamMemberId' does not exist in type '(Without<TeamTechStackItemCreateInput, TeamTechStackItemUncheckedCreateInput> & TeamTechStackItemUncheckedCreateInput) | (Without<...> & TeamTechStackItemCreateInput)'. Did you mean to write 'voyageTeamId'?"